### PR TITLE
Galaxy Force 2, Outrun remake, Sega Rally fix

### PR DESCRIPTION
**Pull request by OctopusButtons**
- New box art file for Galaxy Force 2.
- Better scan for Outrun remake to replace the overly dark shadow scan old one.
- Correction and replace for wrong file type/extension `jpg` on Sega Rally Championship 95 remake (caused by me) to correct `png`.
  - Deleted the old jpg
- Better representations of in-game art, action, atmosphere, for Gradius V and R-Type Final.
  - Correction of those to 512px

### CODE REVIEW

File: Named_Boxarts/Sega Ages 2500 Series Vol. 13 - OutRun (Japan).png
Comment (RobLoach) [inline Named_Boxarts/Sega Ages 2500 Series Vol. 13 - OutRun (Japan).png:1]: Looks like this is pretty desaturated?

File: Named_Snaps/Gradius V (USA).png
Comment (RobLoach) [inline Named_Snaps/Gradius V (USA).png:1]: 513px. Oopsies haha

Comment (OctopusButtons) [reply Named_Snaps/Gradius V (USA).png:1]: Fixed now to 512px.  Whoops.

File: Named_Boxarts/Sega Ages 2500 Series Vol. 13 - OutRun (Japan).png
Comment (OctopusButtons) [reply Named_Boxarts/Sega Ages 2500 Series Vol. 13 - OutRun (Japan).png:1]: This is less desaturated now.  Very few examples exist to be able to judge fadedness vs scan quality/intensity, unfortunately.  Keep in mind the original thumbnail was overly saturated and creates a misleading contrast when comparing.  But anyway new version now.

**Comment by RobLoach** [comment]
thanks for the updates

**Pull request merged**